content/sensu-go/5.14/api/federation.md

```diff
@@ -0,0 +1,408 @@
+	- [`/replicators` (GET)](#replicators-get)
+	- [`/replicators` (POST)](#replicators-post)
+- [The `/replicators/:replicator` API endpoint](#the-replicatorsreplicator-api-endpoint)
+	- [`/replicators/:replicator` (GET)](#replicatorsreplicator-get)
+  - [`/replicators/:replicator` (PUT)](#replicatorsreplicator-put)
+  - [`/replicators/:replicator` (DELETE)](#replicatorsreplicator-delete)
+
+## The `/replicators` API endpoint
+
+**LICENSED TIER**: Unlock the federation API in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][1].
+
+_**NOTE**: The federation API is only accessible for users who have a cluster role that permits access to replication resources._
+
+### `/replicators` (GET)
+
+The `/replicators` API endpoint provides HTTP GET access to a list of replicators.
+
+#### EXAMPLE {#replicator-get-example}
+
+The following example demonstrates a request to the `/replicators` API, resulting in a list of replicators.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/federation/v1/replicators -H "Authorization: Bearer $SENSU_TOKEN"
+[
+  {
+    "entity_class": "agent",
+    "sensu_agent_version": "1.0.0",
+    "system": {
+      "hostname": "sensu-centos",
+      "os": "linux",
+      "platform": "centos",
+      "platform_family": "rhel",
+      "platform_version": "7.4.1708",
+      "network": {
+        "interfaces": [
+          {
+            "name": "lo",
+            "addresses": [
+              "127.0.0.1/8",
+              "::1/128"
+            ]
+          },
+          {
+            "name": "enp0s3",
+            "mac": "08:00:27:11:ad:d2",
+            "addresses": [
+              "10.0.2.15/24",
+              "fe80::f50c:b029:30a5:3e26/64"
+            ]
+          },
+          {
+            "name": "enp0s8",
+            "mac": "08:00:27:9f:5d:f3",
+            "addresses": [
+              "172.28.128.3/24",
+              "fe80::a00:27ff:fe9f:5df3/64"
+            ]
+          }
+        ]
+      },
+      "arch": "amd64"
+    },
+    "subscriptions": [
+      "entity:sensu-centos"
+    ],
+    "last_seen": 1543349936,
+    "deregister": false,
+    "deregistration": {},
+    "user": "agent",
+    "redact": [
+      "password",
+      "passwd",
+      "pass",
+      "api_key",
+      "api_token",
+      "access_key",
+      "secret_key",
+      "private_key",
+      "secret"
+    ],
+    "metadata": {
+      "name": "sensu-centos",
+      "namespace": "default",
+      "labels": null,
+      "annotations": null
+    }
+  }
+]
+{{< /highlight >}}
+
+#### API Specification {#replicators-get-specification}
+
+/replicators (GET)  | 
+---------------|------
+description    | Returns a list of replicators.
+example url    | http://hostname:8080/api/federation/v1/replicators
+pagination     | This endpoint supports pagination using the `limit` and `continue` query parameters. See the [API overview](../overview#pagination) for details.
```

Comment:
Need to confirm this is true (supports pagination)

Reply:
No use case for pagination here, let's not mention it, even though I believe it's technically supported.

Reply:
Thank you! I'll delete it.